docs/api/IoT.js-API-PWM.md

```diff
@@ -17,59 +17,34 @@ The following shows PWM module APIs available for each platform.
 | pwmpin.closeSync | O | O | O |
 
 
-## Contents
-* [Pin](#pin)
-* [PWM](#pwm)
-  * [Constructor](#pwm-constructor)
-    * [`new PWM()`](#pwm-new)
-  * [Prototype methods](#pwm-prototype-methods)
-    * [`pwm.open(configuration[, callback])`](#pwm-open)
-* [PWMPin](#pwmpin)
-  * [Prototype methods](#pwmpin-prototype-methods)
-    * [`pwmpin.setPeriod(period[, callback])`](#pwmpin-period)
-    * [`pwmpin.setPeriodSync(period)`](#pwmpin-period-sync)
-    * [`pwmpin.setFrequency(frequency[, callback])`](#pwmpin-frequency)
-    * [`pwmpin.setFrequencySync(frequency)`](#pwmpin-frequency-sync)
-    * [`pwmpin.setDutyCycle(dutyCycle[, callback])`](#pwmpin-dutycycle)
-    * [`pwmpin.setDutyCycleSync(dutyCycle)`](#pwmpin-dutycycle-sync)
-    * [`pwmpin.setEnable(enable[, callback])`](#pwmpin-enable)
-    * [`pwmpin.setEnableSync(enable)`](#pwmpin-enable-sync)
-    * [`pwmpin.close([callback])`](#pwmpin-close)
-    * [`pwmpin.closeSync()`](#pwmpin-close-sync)
-
-
-### `pin` <a name="pin"></a>
-* On Linux, `pin` is a pwm number which is 0 or 1.
-* On Nuttx, you have to know pin name. The pin name is defined in target board module. For more module information, please see below list.
-  * [STM32F4-discovery](../targets/nuttx/stm32f4dis/IoT.js-API-Stm32f4dis.md#pwm-pin)
-
-
-## Class: PWM <a name="pwm"></a>
-
+### `pin`
 
-## Constructor <a name="pwm-constructor"></a>
 
+## Class: PWM
 
-### `new PWM()` <a name="pwm-new"></a>
+### new PWM()
 
-Returns a new PWM object which can open PWM pin.
+Returns a new PWM object which can open a PWM pin.
 
 
-## Prototype methods <a name="pwm-prototype-methods"></a>
-
-
-### `pwm.open(configuration[, callback])` <a name="pwm-open"></a>
-* `configuration <Object>`
-  * `pin <Number>`, mandatory configuration
-  * `chip <Number>`, chip number, Default: 0, only Linux configuration
-  * `period <Number>`, seconds(positive integer) 
-  * `frequency <Number>`, Hz(positive integer)
-  * `dutyCycle <Number>`, between 0 and 1
-* `callback <Function(err: Error | null)>`
+### pwm.open(configuration[, callback])
+* `configuration` {Object} Configuration object which can have the following entries
+  * `pin` {number} The specific pin to use with this PWM object (mandatory configuration).
+  * `chip` {number} The PWM chip number (only on Linux). **Default:** `0`.
```

Comment:
Don't mix types. This should be Specify the PWM number. Or the above: The pin index of the PWM object.